code/Cinema.csproj

```diff
@@ -30,26 +30,26 @@
 	</ItemGroup>
 
   <PropertyGroup>
-	<OutputPath>D:\SteamLibrary\steamapps\common\sbox/.vs/output/</OutputPath>
-	<DocumentationFile>D:\SteamLibrary\steamapps\common\sbox/.vs/output/cinema.xml</DocumentationFile>
+	<OutputPath>..\..\..\..\Program Files\Steam\steamapps\common\sbox/.vs/output/</OutputPath>
```

Comment:
I'm thinking we maybe just remove the csproj file all together since this only solves it if you have sbox located in particular place too.